Feature/7

bestrizhiy opened the pull request:
Добрый вечер, Марина Алексеевна!
Это Тяжко Даниил - студент ТКИ-511 , загрузил курсовой проект за прошлый семестр, проверьте пожалуйста.

bestrizhiy committed:
Delete Task1

bestrizhiy committed:
Загрузил курсовой проект

bestrizhiy committed:
Update запрос1.sql

bestrizhiy committed:
Update запрос2.sql

bestrizhiy committed:
Update запрос3.sql

bestrizhiy committed:
Update запрос4.sql

bestrizhiy committed:
Update запрос5.sql

MarinaPaley commented on Курсач/SQL Server Scripts1/SQL Server Scripts1/DB.sql:
```diff
@@ -0,0 +1,253 @@
+  , Medical_Insurance NVARCHAR(50) NOT NULL UNIQUE
+);
+
+CREATE TABLE Countries
+(
+	ID INTEGER PRIMARY KEY
+  , [Name] NVARCHAR(50) NOT NULL UNIQUE
+);
+
+CREATE TABLE Cities
+(
+	ID INTEGER PRIMARY KEY
+  , [Name] NVARCHAR(50) NOT NULL UNIQUE
+);
```
города должны иметь связь один-ко-многим со страной

bestrizhiy replied:
залил исправленный вариант

bestrizhiy committed:
Исправленный вариант курсовой

Добрый ночи, Марина Алексеевна!

Залил исправленный курсовой проект за прошлый семестр.

Проверьте, пожалуйста.

bestrizhiy committed:
Delete Курсач/SQL Server Scripts1/курсач исправлено directory

bestrizhiy committed:
Исправленный курсовой проект

Добрый ночи, Марина Алексеевна!

Залил исправленный курсовой проект за прошлый семестр.

Проверьте, пожалуйста.
